geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ConverterIngest.scala

```diff
@@ -104,8 +98,11 @@ class ConverterIngest(dsParams: Map[String, String],
               toWrite.getUserData.put(Hints.USE_PROVIDED_FID, java.lang.Boolean.TRUE)
               try {
                 fw.write()
+                written.incrementAndGet()
```

Comment:
ah, good point... but yeah, that would be hard to track, b/c you don't even know when it flushes, right?